client/src/app/app.component.ts

```diff
@@ -93,6 +93,15 @@ export class AppComponent implements OnInit, AfterViewInit {
     this.pluginService.initializeCustomModal(this.customModal)
   }
 
+  @HostListener('scroll')
+  onContentScroll() {
+    const dropdown = document.querySelector('body > .dropdown')
+
+    if (dropdown) {
+      document.body.removeChild(dropdown)
+    }
+  }
```

Comment:
WDYT @Chocobozzz?

Reply:
Maybe it would be better to listen scroll event only when an `action-dropdown` is opened and destroy the event after removing the append dropdown to the body ?